tictactoe/eacs/build.gradle

```diff
@@ -112,3 +117,6 @@ task rollForwardTest(type: Test) {
         "com.anaplan.engineering.azuki.persistence.verification.instance.jar": sampleImplementationV2ClasspathJar.archiveFile.get().asFile.absolutePath,
     ]
 }
+
+check.dependsOn(rollBackTest)
+check.dependsOn(rollForwardTest)
```

Comment:
The default impls and these dependencies are to ensure all tests are run on full build -- noticed they weren't running and so I had unexpected confidence in change.